evm/vm/evm.py

```diff
@@ -125,10 +106,10 @@ def get_vm_class_for_block_number(cls, block_number):
         """
         Return the vm class for the given block number.
         """
-        range = find_range(tuple(cls.vms_by_range.keys()), block_number)
-        base_vm_class = cls.vms_by_range[range]
-        vm_class = base_vm_class.configure(db=cls.db)
```

Comment:
Apart from that it seems to [work in all situations ;-)](https://travis-ci.org/pipermerriam/py-evm/builds/247939268). Should I squash those commits or would you rather keep them separate?